### ISSUE TITLE
URL for thumbnails in mail notification doesn't populate the method variable with scale from notif.html

### ISSUE THREAD

**Issue by tgurr** [matrix-org/synapse#12506]
### Description

Link for thumbnails in mail notification doesn't populate the method variable resulting in `errcode: M_NOT_FOUND`

Inspecting the mails HTML code reveals the following link to the preview image:

`https://matrix.domain.corporate/_matrix/media/v1/thumbnail/domain.corporate/DAqDEJPvGLMpWPidjpgBfgaU?width=640&height=480&method=`

which results in 

`{"errcode":"M_NOT_FOUND","error":"Not found [b'domain.corporate', b'DAqDEJPvGLMpWPidjpgBfgaU']"}`

when opened in a browser window.

When manually adding the `scale` property which is present in `notif.html` mail template, code

```
                    {%- elif message.msgtype == "m.image" and message.image_url %}
                        <img src="{{ message.image_url|mxc_to_http(640, 480, scale) }}" />
```
to the URL like

`https://matrix.domain.corporate/_matrix/media/v1/thumbnail/domain.corporate/DAqDEJPvGLMpWPidjpgBfgaU?width=640&height=480&method=scale`

the preview thumbnail is shown properly.

### Steps to reproduce

- enable mail notifications
- be unavailable
- have someone send you a message with an image
- see the preview/link in the sent notification mail not working

### Version information

- **Homeserver**: local instance

If not matrix.org:

- **Version**: {"server_version":"1.57.0","python_version":"3.9.10"}

- **Install method**: package manager

- **Platform**: Exherbo Linux / virtual machine

**Comment by clokep** [comment]
> When manually adding the `scale` property which is present in `notif.html` mail template, code
> 
> ```
>                     {%- elif message.msgtype == "m.image" and message.image_url %}
>                         <img src="{{ message.image_url|mxc_to_http(640, 480, scale) }}" />
> ```

I think this can be fixed by putting single quotes around the scale (currently JInja is treating it as a variable, which doesn't exist, so falls back to an empty string). If you change `notif.html` to:

```jinja
                    {%- elif message.msgtype == "m.image" and message.image_url %}
                        <img src="{{ message.image_url|mxc_to_http(640, 480, 'scale') }}" />
```

does it work?

**Comment by clokep** [comment]
Great! Should be fixed by #12510 then!
